lib/member_page.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class String
+  def tidy
+    gsub(/[[:space:]]+/, ' ').strip
+  end
+end
+
+class MemberPage < Scraped::HTML
+  decorator Scraped::Response::Decorator::AbsoluteUrls
+  # TODO: Add contact info
+  # Issue:
+  # https://github.com/everypolitician/everypolitician-data/issues/22638
+  field :id do
+    url_slug.split('-').last
+  end
+
+  field :name do
+    url_slug.split('-')[0..-2]
+            .map(&:upcase)
+            .join(' ')
+  end
+
+  field :name__ru do
+    noko.at_css('.dep_info h1').text.tidy
+  end
+
+  field :term_id do
```

Comment:
Is the term_id actually on this page somewhere? If so, you should extract it from there. If not, you should probably just add this separately when gathering the data to write to Morph. Ideally these classes would be a representation of the page itself, without adding extra information (yes, we've done that in other scrapers, but we're moving away from that as the dust settles and we work out the best way for these to function). 

NB: When writing to Morph (which is primarily for EP, whereas these classes should be more generic than that), you'll want to simply call this `term`, not `term_id`): http://docs.everypolitician.org/submitting.html